pom.xml

```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
-            <version>4.0.1</version>
+            <version>4.0</version>
```

Comment:
I think 4.0 has a bug which is why we were using 4.0.1.

Reply:
This change was made to align with the Open Liberty starter application that you can generate. I can change that back to 4.0.1